### ISSUE TITLE
Unable to save labkit output from command line

### ISSUE THREAD

**Issue by YuqiTianCIVM** [juglab/labkit-ui#111]
We are able to save the output from labkit when starting Fiji normally and running the macro, but unable to save result from command line Fiji labkit with either -macro or -batch. 

Github doesn't support arbitrary files, so I cannot include the classifier. Any classifier should work to demonstrate the issue. 

The example is as below:
```
folder="S:/Tif_in/"
output="S:/segmented_out/"
processed="S:/tif_processed/"
s="s:/project.classifier";
File.makeDirectory(output);
File.makeDirectory(processed);
N=1
for (i = 0; i < N; i++) {
    open(folder+"Region_"+i+".tif");
    run("Segment Image With Labkit", "segmenter_file="+s+" use_gpu=false");
    print("Images open: "+nImages);
    close("Region_"+i+".tif");
    saveAs("Tiff", output + "segmentation_" + i + ".tif");
    run("Make Binary", "method=Default background=Default calculate");
    run("Watershed","stack");
    run("Connected Components Labeling", "connectivity=6 type=[16 bits]");
    saveAs("Tiff", processed + "morpho_" + i + ".tif");
    close("segmentation_" + i + ".tif");
    close("MASK_segmentation_" + i + ".tif");
    close("morpho_" + i + ".tif");
}
exit();
```
![Region_0](https://github.com/juglab/labkit-ui/assets/50126705/629ee07a-0ee2-4dec-a915-356a2a5d5396)

**Comment by maarzt** [comment]
@YuqiTianCIVM Thank you for the detailed description of the problem. I was able the reproduce it. The cause of the issue is very likely some bug imagej-legacy. That said, I was able to implement a workaround.

You need to active the Labkit update site to get the latest version of Labkit.

Then you will find two new entries in the Fiji menu: "Plugins > Labkit > Macro Recordable > Segment Image With Labkit (IJ1)" and "Plugins > Labkit > Macro Recordable > Calculate Probability Map With Labkit (IJ1)".

Your script should work if you use these new commands. Here is an updated version of your script that worked for me:
(I had to use different command for thresholding and watershed to actually get it working.)
```
folder="S:/Tif_in/"
output="S:/segmented_out/"
processed="S:/tif_processed/"
s="S:/project.classifier";
File.makeDirectory(output);
File.makeDirectory(processed);
N=1
for (i = 0; i < N; i++) {
    open(folder+"Region_"+i+".tif");
    run("Segment Image With Labkit (IJ1)", "segmenter_file="+s+" use_gpu=false");
    print("Images open: "+nImages);
    close("Region_"+i+".tif");
    saveAs("Tiff", output + "segmentation_" + i + ".tif");
    run("Convert to Mask", "method=Otsu background=Dark calculate create");
    selectImage("MASK_segmentation_" + i + ".tif");
    run("Distance Transform Watershed 3D", "distances=[Borgefors (3,4,5)] output=[16 bits] normalize dynamic=2 connectivity=6");
    selectImage("MASK_segmentation_" + i + "dist-watershed");
    saveAs("Tiff", processed + "morpho_" + i + ".tif");
    close("segmentation_" + i + ".tif");
    close("MASK_segmentation_" + i + ".tif");
    close("morpho_" + i + ".tif");
}
exit();
````

**Comment by maarzt** [comment]
This is now fixed in 214ff98ba57917b4f4fcbcfa46f3dbd33f114e70 and will be part of the next Labkit release
